README.md

```diff
@@ -1,85 +1,99 @@
-# helmbin
+# Helmbin
 
-Embeds Kubernetes and Helm charts as a single installable binary.
+Helmbin is a command-line tool that enables users to render and run Helm Charts within a Virtual Machine.
+
+## Introduction
+
+Helmbin embeds Kubernetes and Helm charts in a single installable binary. It is designed to simplify the process of rendering and deploying Helm Charts by providing a lightweight and isolated environment that can then be installed in a Virtual Machine. With Helmbin, you can quickly deploy Helm-based applications without the need for complex setup or dependencies.
+
+## Installation
+
+To install Helmbin, follow these steps:
+
+1. Download the latest release from the [Helmbin releases page](https://github.com/replicatedhq/helmbin/releases).
```

Comment:
for the future. i would like to add an install.sh similar to [this](https://github.com/k3s-io/k3s/blob/master/install.sh)